documentation/book/assembly-cluster-operator.adoc

```diff
@@ -9,18 +9,21 @@
 
 include::common/attributes.adoc[]
 
-[id='deploying-cluster-operator-{context}']
-= Deploying the Cluster Operator
-
-{ProductName} uses the Cluster Operator to deploy and manage Kafka (including Zookeeper) and Kafka Connect
-clusters.
-The Cluster Operator is deployed as a process running inside of the {ProductPlatformName} cluster.
-To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created within the {ProductPlatformName} cluster.
-Based on the information in that `Kafka` resource,
+[id='cluster-operator-{context}']
+= Cluster Operator
+
+{ProductName} uses the Cluster Operator to deploy and manage Kafka (including Zookeeper) and Kafka Connect clusters.
+The Cluster Operator is deployed inside of the
+ifdef::Kubernetes[Kubernetes or]
+OpenShift cluster.
+To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created within the
+ifdef::Kubernetes[Kubernetes or]
+OpenShift cluster.
+Based on the information in the `Kafka` resource,
```

Comment:
It declarative management. Would _declaration_ work here instead of _information_? They way this works is that the user declares what she/he wants and the CO just implements it.